src/containers/App/test/App.test.jsx

```diff
@@ -270,6 +262,12 @@ describe('App container', () => {
           event: 'screen_view',
           network: 'mainnet',
         },
+        {
```

Comment:
I'm not sure why these test outputs changed. This should be investigated before this PR is merged.